### ISSUE TITLE
Suggest cakedc/migrations depedency for CakePHP 2.x compatible version

### ISSUE THREAD

**Issue by liviascapin** [orca-services/cakephp-heartbeat#8]
To be able to use the DBUpToDate Sensor one needs to add cakedc/migrations as dependency.
Currently we do not (and should not) enforce this dependency but at least we should suggest it in the composer JSON.

**Comment by ravage84** [comment]
(Speaking of the CakePHP 3.x compatible version of the plugin & the new Migrations plugin, not the one from CakeDC)

@TekkCraft  and I have discussed this today related to the issues with updating to CakePHP 3.7 #17,  #18, & #19.

We came to the conclusion that the actual problem is that this project contains a sensor which depends on a package which all other sensors do not depend on. This means either the list of dependencies lie (as currently the case) or that we have to add it as full dependency to rightfully fullfill this dependency.

Both ways aren't nice. The best solution from a dependency management standpoint would be to extract this one sensor to a separate package, that would have the Migrations plugin as dependency. This would increase the maintenance, though.
